src/endless/EndlessUsbToolDlg.cpp

```diff
@@ -1553,6 +1556,34 @@ void CEndlessUsbToolDlg::ErrorOccured(ErrorCause_t errorCause)
     }
 
     ChangePage(_T(ELEMENT_ERROR_PAGE));
+
+	Analytics::instance()->exceptionTracking(ErrorCauseToStr(errorCause), FALSE);
```

Comment:
Whilst every error has a "Retry" button they are in effect fatal unless we have a known remedy - can we set this a little more consistently: I would say the non-fatal ones are download timeout & verification failure because we "expect" to work again, and with a USB write failure we suggest using a different USB, but everything else is basically a fatal error because we have no reasonable explanation, or expectation for it to work differently/better a 2nd time.